src/ngraph/opsets/opset.cpp

```diff
@@ -85,3 +85,22 @@ const ngraph::OpSet& ngraph::get_opset2()
     }
     return opset;
 }
+
+const ngraph::OpSet& ngraph::get_opset3()
+{
+    static std::mutex init_mutex;
+    static bool opset_is_initialized = false;
```

Comment:
I'm not sure that this extra bool is needed. Since C++11 the Meyer's singleton is guaranteed to be safe with just the object being static.

Reply:
Me neither. I've followed convention from: https://github.com/NervanaSystems/ngraph/pull/4483